generators/csharp/codegen/src/context/convertExampleTypeReferenceToTypeReference.ts

```diff
@@ -0,0 +1,202 @@
+import { assertNever } from "@fern-api/core-utils";
```

Comment:
It looks like this file is duplicated in `ir-utils` - did you mean to import that version here instead?